06_modules_lab/03.py

```diff
@@ -1,11 +1,12 @@
-""" Write a program that searches current working directory
-for files larger than 1MB. Every time you find such a file print
-its name to the user.
-
-- When the program finds a large file. It should ask the user
-  a message asking if she wants to delete it, and delete the
-  file if requested
-
-- Take threshold and path as command line arguments
-"""
-
+"""remove big file"""
+import os,sys
+if len(sys.argv) < 2 :
+    path = 'c:\python/week 2'

```

Comment:
When using paths it's recommended to prefix them with r. Here's why:

This works ok:

```
>>> x = 'c:\python'
>>> print(x)
c:\python
```

But this breaks:

```
>>> x = 'c:\bython'
>>> print x
cython
```

The reason is the sign \ has different meaning depending on what comes next. `\p` means nothing so it's a normal \, but `\b` is backspace which deletes the previous character (the colon) so we're left with `cython`.

Adding r prefix cancels this behaviour so \ is always a \:

```
>>> x = r'c:\python'
>>> print x
c:\python
>>> x = r'c:\bython'
>>> print x
c:\bython
```